__docs__/wonder-blocks-form/text-area-variants.stories.tsx

```diff
@@ -10,8 +10,6 @@ import {TextArea} from "@khanacademy/wonder-blocks-form";
 /**
  * The following stories are used to generate the pseudo states for the
  * TextArea component. This is only used for visual testing in Chromatic.
- *
- * Note: Error state is not shown on initial render if the TextArea value is empty.
```

Comment:
Since we can explicitly put the component in an error state, the variants stories can properly show the error styling in all cases now!